pythran/pythonic/numpy/stack.hpp

```diff
@@ -29,16 +26,32 @@ namespace numpy
         sutils::push_front_t<decltype(shape), long> new_shape;
         // Assign the values array to a tuple created from dim_array. I'm sure there's a shortcut for these two operations.
         new_shape.values = types::make_tuple(dim_array);
-      
+        
         // Create a new empty list.
-        types::list<types::ndarray<typename E::dtype, types::array<long, E::value+1>>> bi(0);
+        types::list<types::ndarray<dtype, types::array<long, N+1>>> bi(0);
         // Push the resized arrays into the list.
-        for (i=0;i<args.size();i++) {
-            bi.push_back(args[i].reshape(new_shape));
+        for (auto item : args) {
+            bi.push_back(item.reshape(new_shape));
         }
         // Call concatenate on this list.
         return concatenate(bi,axis);
     }
+    
+    template <class E>
+    types::ndarray<typename E::dtype, types::array<long, E::value+1>>
+    stack(types::list<E> const &args, long axis)
+    {
+        return _stack<types::list<E>, typename E::dtype, E::value>(args,axis);
+    }
+
+    
+    template <class E, size_t M>
+    types::ndarray<typename E::dtype, types::array<long, E::value+1>>
+    stack(types::array<E, M> const &args, long axis)
+    {
+        return _stack<types::array<E, M>, typename E::dtype, E::value>(args,axis);
+    }
+    
```

Comment:
I think I handled that case in my PR too :-)

Reply:
ah ah ah of course of course. Took you 1/2 a second I'm sure! :)